community/modules/scripts/spack/outputs.tf

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+output "spack_deps_runner" {
+  description = <<-EOT
+    Runner to install dependencies for spack using an ansible playbook. The
+    startup-script module will automatically handle installation of ansible.
+    - id: example-startup-script
+      source: modules/scripts/startup-script
+      settings:
+        runners:
+        - $(your-spack-id.spack_deps_runner)
+    ...
+    EOT
+  value       = local.spack_deps_runner
+}
+
+/* Startup script runner that performs full installation of spack, given
+ * dependencies are installed already. For more information on the individual
+ * steps of the installation, see the fine-grained runners below.
+ */
+output "spack_full_install_runner" {
+  description = <<-EOT
+    Runner that incorporates the contents of the following other runners, executed
+    in this order: spack_clone_runner, spack_commands_runner,
+    spack_compiler_runner, spack_packages_runner.
+
+    Usage:
+    - id: example-startup-script
+      source: modules/scripts/startup-script
+      settings:
+        runners:
+        - $(your-spack-id.spack_full_install_runner)
+    ...
+    EOT
+  value       = local.spack_full_install_runner
+}
+
+output "spack_setup_runner" {
+  description = <<-EOT
+    Adds Spack setup-env.sh script to /etc/profile.d so that it is called at
+    shell startup. Among other things this adds Spack binary to user PATH.
+    EOT
+  value = {
+    type        = "data"
+    destination = "/etc/profile.d/spack.sh"
```

Comment:
Does the name of this file matter. As it stands now the logging for this is as follows:
```
startup-script: Info [9734]: === start executing runner: spack.sh ===
startup-script: Info [9734]: === spack.sh finished with exit_code=0 ===
```

If we named the file spack_env_setup.sh that would translate to the logging and be a bit more clear.

Reply:
I do not think the name of the file matters. So, anything to help make it more clear would be fine.